src/ui.data-table/UTable.vue

```diff
@@ -557,6 +558,16 @@ watch(
     tableRows.value = tableRows.value.map((row) => syncRowCheck(row, selectedRows.value));
   },
 );
+watch(
+  () => props.rows,
+  () => {
+    if (!rowsHasId(props.rows)) {
+      // eslint-disable-next-line no-console
+      console.warn("[Vueless][UTable]: Each table row must have unique id.");
```

Comment:
maybe it's better to assign the `id` by yourself if it not passed by user. There are a lot of cases where `id` no need to add.

Reply:
We should consider this and work with keys in general. I still suggest to move it into once reserved property. Let's leave warning for now and move it to another task for discussion.